app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -571,6 +575,48 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         hideProgress()
     }
 
+    private fun zoomToShowRoute(route: Route) {
+        //Get size of display For calculations
+        var mdisp = getWindowManager().getDefaultDisplay()
+        var mdispSize = Point()
+        mdisp.getSize(mdispSize)
+        var width = mdispSize.y
+
+        var geometry = route?.getGeometry()
```

Comment:
Most `?.` access in this method can be replaced with `.` since values are not nullable.